### PR TITLE
README: fix grammar

### PR DESCRIPTION
**Pull request by Sumindar**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ to see the current progress of all translations.
 - [kb](https://github.com/gnebbia/kb)
   is a minimalist command-line knowledge base manager.
   kb can be used to organize your notes and cheatsheets in a minimalist
-  and clean way. It supports also non-text files.
+  and clean way. It supports  non-text files also.
```
Comment (navarroaxel) [inline README.md:197]: double space here
```suggestion
  and clean way. It supports non-text files also.
```

Comment (CleanMachine1) [reply README.md:197]: The also would go at the beginning on the sentence otherwise its less formal

Comment (marchersimon) [reply README.md:197]: I also think it rather should be
```suggestion
  and clean way. It also supports non-text files.
```

Comment (sbrl) [reply README.md:197]: @marchersimon's suggestion reads most naturally to me.

**Review by sbrl — APPROVED** [review]
Thanks, @Sumindar!

File: README.md (hunk 1 above)
Comment (sbrl) [reply README.md]: Leaving unresolved on purpose.

**Review by marchersimon — APPROVED** [review]
Thanks

**Pull request merged**